src/javascript/app/base/header.js

```diff
@@ -63,10 +64,12 @@ const Header = (() => {
 
     const loginOnClick = (e) => {
         e.preventDefault();
+        Defaults.remove('market', 'underlying');
         Login.redirectToLogin();
     };
 
     const logoutOnClick = () => {
+        Defaults.remove('market', 'underlying');
```

Comment:
can we move this to a separate function, something like `clearTradeParams` and add a comment to explain why we need to clear them?

Reply:
Done